bin/clever.js

```diff
@@ -212,6 +217,23 @@ function run() {
     parser: Parsers.date,
     description: "Fetch logs after this date (ISO8601)"
   });
+  var listAllWebhooksOption = cliparse.flag("list-all", {
+    description: "List all webhooks for your user or for an organisation with the `--org` option"
+  });
+  var webhookFormatOption = cliparse.option("format", {
+    metavar: "format",
+    default: "raw",
+    description: "Format of the body sent to the webhook ('raw', 'slack', or 'flowdock')"
```

Comment:
nope, definitely sent to something but I would say "to the url" or simply "format of the body"